.storybook/storybook-styles.css

```diff
@@ -1,5 +1,4 @@
 .docs-story > div:first-child {
```

Comment:
Since we control what customers see now we only have to worry about the representation of stories in docs. And not all of the story examples